scripts/write_deployinst_input.py

```diff
@@ -392,15 +393,15 @@ def main(csv_file, reactor_template, deployinst_template,
         data['entry_time'] = entry_time
         data['lifetime'] = lifetime
     # renders reactor / region / input file. Confesses imperfection.
-    reactor_render(dataset, reactor_template, reactor_output)
+    reactor_render(dataset, reactor_template, 'reactor_output.xml.in')
```

Comment:
Gotcha. This is fine as is for now. Someday perhaps it will become necessary to make it more general. If so, it will be clear how at that point. If not, then it will be fine staying as is!